resource/validate.go

```diff
@@ -25,6 +25,37 @@ const (
 	SKIP
 )
 
+var humanOutcomes map[int]string = map[int]string{
+	SUCCESS: "pass",
+	FAIL:    "fail",
+	SKIP:    "skip",
+}
+
+func HumanOutcomes() map[int]string {
+	return humanOutcomes
+}
+
+func Resources() []string {
```

Comment:
This is a hack and I'm about to open a different PR that will let this be replaced by the same pattern that the `outputs` package uses.

Reply:
That other PR is #611.